articles/continuous-system-integration.md

```diff
@@ -0,0 +1,41 @@
+---
+authorName: Erik Steiner
+authorGithubUsername: estfra
+issue: 63
+title: Successful Agile System Development with Continuous System Integration
+---
+# {{page.title}}
+
+Continuous integration (CI) is the core of successful agile software development. In consequence, successful agile systems development, e.g., development of a medical device, must be based on a CI infrastructure on the system level.
```

Comment:
Der Kern agiler Softwarentwicklung ist wohl nicht für jeden die CI. Wie wärs mit "CI is a core practice of successful ..."

"In consequence" im zweiten Satz ist etwas schwer verständlich. In Konsequenz zu was? 

Als Einstieg in die Problemstellung könnte man vielleicht fragen: Does this apply to agile systems development (i.e. development of a medical device) as well? 

Oder starten mit einer vielleicht provokanten Äusserung: A practice which makes agile systems development (i.e. development of a medical device) successfull as well!

Das Beispiel i.e. development of a medical device finde ich übrigens sehr gut. Denn "systems" überliest man schnell.

Reply:
Vielen Dank für die kreativen Hinweise. Fragen mag ich nicht so gerne als Stilmittel verwenden. Aber die provokante Äusserung gefällt mir sehr gut. :-) Ich denke, die Einleitung wird damit schlanker und verständlicher.